server/controllers/article.controller.js

```diff
@@ -1,8 +1,8 @@
 import db from '../models';
-import slug from '../utils/slugify.utils';
+import slugify from '../utils/slugify.utils';
 import Response from '../utils/response.util';
 
-const { Article } = db;
+const { Article, tag, articletag } = db;
```

Comment:
There is inconsistency in table naming @davidokonji

Reply:
Yes would effect the change